css/screen/medium-screen..css

```diff
@@ -0,0 +1,74 @@
+.body {
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.stories::before {
+  display: none;
+}
+
+.stories {
+  padding-top: 0;
+}
+
+.d-sm-block {
+  display: none;
+}
+
+.d-md-block {
+  display: block;
+}
+
+.login {
+  background-image: none;
+  font-size: 15px;
+  line-height: 30px;
+  padding: 0 15px;
+  margin: 5px 0;
+  cursor: pointer;
+  text-transform: uppercase;
+  color: #fff;
+  border-radius: 24px;
+  font-weight: bold;
+}
+
+.image1,
+.image3,
+.image2,
+.image6 {
+  width: 63px;
+}
+
+.image4,
+.image5 {
+  width: 65px;
+}
```

Comment:
Why are these class selector having a different width? 🤔

Reply:
The image seems to have different sizes, I will see to making it have the same width, thanks a lot for your review!